dot/digest/block_import.go

```diff
@@ -69,15 +69,23 @@ func (h *BlockImportHandler) handleConsensusDigest(d *types.ConsensusDigest, hea
 			return err
 		}
 
-		return h.grandpaState.HandleGRANDPADigest(header, data)
+		err = h.grandpaState.HandleGRANDPADigest(header, data)
+		if err != nil {
+			return fmt.Errorf("while handling grandpa digest: %w", err)
+		}
+		return nil
```

Comment:
can we return nil at the end of this func, instead of returning nil from every case?